src/main/kotlin/rys/ajaxpetproject/repository/ChatRepository.kt

```diff
@@ -0,0 +1,13 @@
+package rys.ajaxpetproject.repository
+
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.repository.MongoRepository
+import org.springframework.stereotype.Repository
+import rys.ajaxpetproject.model.MongoChat
+
+@Repository
+interface ChatRepository : MongoRepository<MongoChat, ObjectId> {
+     fun findChatById(id:ObjectId) : MongoChat?
+     fun findAllBy() : List<MongoChat>?
+
+}
```

Comment:
```suggestion
     fun findAllBy() : List<MongoChat>?
}
```

can null list be returned?

Reply:
Done